.github/test/mocks.js

```diff
@@ -44,6 +44,7 @@ export function createMockCore() {
   return {
     debug: vi.fn(console.debug),
     info: vi.fn(console.log),
+    isDebug: vi.fn().mockReturnValue(true),
```

Comment:
Enable debug logging by default in unit tests (viewable in test results window in VSCode)